Frends.SFTP.ListFiles/Frends.SFTP.ListFiles/Enums/IncludeType.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// Pragma for self-explanatory enum attributes.
 #pragma warning disable 1591
 
-namespace Frends.SFTP.ListFiles.Enums;
+namespace Frends.SFTP.ListFiles.Definitions;
```

Comment:
Namespace should be Enums, since the files are in Enums-folder, not in Definitions.